Description on namespaces that users might want to change (#28060)
openshift/openshift-docs: serverless/networking/serverless-ossm.adoc

apiVersion: maistra.io/v1
kind: ServiceMeshMemberRoll
metadata:
  name: default <1>
  namespace: istio-system
spec:
  members:
    - default <2>
----

> <1> A name of {ProductShortName} Member Roll object, that isn't connected to namespace from <2>

Reviewer: ```suggestion
<1> The name of a {ProductShortName} `ServiceMeshMemberRoll` object, that isn't connected to a namespace listed in the `spec` field.
```

> <2> A list of namespaces you like to integrate {ServerlessProductName} with {ProductName}

Reviewer: ```suggestion
<2> A list of namespaces where {ServerlessProductName} will be integrated with {ProductName}.
```

Reviewer 2: +1
I think this is one of the rare places where using future tense makes the meaning clearer.

  policyTypes:
  - Ingress
----

> <1> NetworkPolicy must be created for every namespace used in {ProductShortName} Member Roll object

Reviewer: ```suggestion
<1> A network policy must be created for every namespace included in a `ServiceMeshMemberRoll` object.
```
